From 2ae14840b9972ae6514c63f694b9293c7008e700 Mon Sep 17 00:00:00 2001
From: hartleys <hartleys@visionengravers.com>
Date: Tue, 24 Apr 2012 14:38:37 +0000
Subject: [PATCH 500/938] crush: include header for global symbols
Patch-mainline: 3.10-rc2
References: fate#312983

Include the header to pickup the definitions of the global symbols.

Quiets the following sparse warnings:

warning: symbol 'crush_find_rule' was not declared. Should it be static?
warning: symbol 'crush_do_rule' was not declared. Should it be static?

Signed-off-by: H Hartley Sweeten <hsweeten@visionengravers.com>
Cc: Sage Weil <sage@newdream.net>
Cc: "David S. Miller" <davem@davemloft.net>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Danny Al-Gaaf <dalgaaf@suse.de>

---
 net/ceph/crush/mapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/ceph/crush/mapper.c b/net/ceph/crush/mapper.c
index 11cf352..2fbe092 100644
--- a/net/ceph/crush/mapper.c
+++ b/net/ceph/crush/mapper.c
@@ -20,6 +20,7 @@
 
 #include <linux/crush/crush.h>
 #include <linux/crush/hash.h>
+#include <linux/crush/mapper.h>
 
 /*
  * Implement the core CRUSH mapping algorithm.
-- 
1.8.3

